build/release/containers.sh

```diff
@@ -65,28 +84,57 @@ publish() {
     publish_artifact $os $arch rook/rookd
 }
 
-cleanup() {
+promote_artifact() {
+    local os=$1
+    local arch=$2
+    local repo=$3
+
+    local src=${registry}$(get_image_name $os $arch $repo ${RELEASE_VERSION})
+    local dst1=${registry}$(get_image_name $os $arch $repo ${RELEASE_CHANNEL}-latest)
+    local dst2=${registry}$(get_image_name $os $arch $repo ${RELEASE_CHANNEL}-${RELEASE_VERSION})
+
+    echo promoting container ${src} to ${dst1} and ${dst2}
+    docker pull ${src}
+    docker tag ${src} ${dst1}
+    docker tag ${src} ${dst2}
+    docker push ${dst1}
+    docker push ${dst2}
+}
+
+promote() {
     local os=$1
     local arch=$2
 
     [[ ${os} == "linux" ]] || return 0
 
-    docker rmi ${registry}${tag} || true
+    promote_artifact $os $arch rook/rookd
 }
 
-publish_artifact() {
+cleanup_artifact() {
     local os=$1
     local arch=$2
     local repo=$3
-    local version=${RELEASE_VERSION}
 
-    tag=${repo}-${arch}:${version}
-    if [[ ${arch} == "amd64" ]]; then
-        tag=${repo}:${version}
-    fi
+    local src=${registry}$(get_image_name $os $arch $repo ${RELEASE_VERSION})
+    echo removing docker image ${src}
+    docker rmi ${src} || true
```

Comment:
what's the reason for the OR-list here that uses true?  is this a way to always make the operation succeed so we don't bail out due to `bash -e`?